data/abbreviations

```diff
@@ -1,28 +1,54 @@
-Mr.
-Mrs.
-Ms.
+al.
+Apr.
+Aug.
+Bros.
 Capt.
+cf.
+ch.
+Co.
+Corp.
+cp.
+Dec.
 Dr.
-Prof.
+e.g.
+esp.
+etc.
+Feb.
 Gen.
 Gov.
-e.g.
-i.e.
-Sgt.
-St.
-vol.
-vs.
-Sen.
-Rep.
-Pres.
 Hon.
-Rev.
-Ph.D.
-M.D.
+i.e.
+Inc.
+Jan.
+Jr.
+Jul.
+Jun.
+Ltd
+Ltd.
 M.A.
+M.D.
+Mar.
+Mr.
+Mrs.
+Ms.
+No.
+Nov.
+Oct.
+Okt.
 p.
+Ph.D.
+PhD.
```

Comment:
I've only seen *Ph.D.* or *PhD*, but not *PhD.*. Is this commonly used?

Reply:
I don't think it's commonly used so I removed it. I also removed the german abbreviations except `Sept.`.